src/Plots.jl

```diff
@@ -122,9 +122,11 @@ export
     mov,
     mp4,
     webm,
+    apng,
```

Comment:
I'm puzzled: not all file methods are exported, and exporting lots of symbols from a package is pretty bad (see https://github.com/JuliaPlots/Plots.jl/issues/3626).

@BeastyBlacksmith, what do you think ?

Reply:
I think this new feature may not be used by a vast majority of users. We may leave it unexported ?

Reply:
Yes, I think it would be better that way.

Reply:
Done